app/src/main/kotlin/com/intive/picover/profile/view/ProfileScreen.kt

```diff
@@ -78,3 +103,74 @@ private fun DeleteAccountDialog(
 		text = { Text(stringResource(R.string.DeleteAccountConfirmationDialogDescription)) },
 	)
 }
+
+@Composable
+private fun ProfileSegment(
+	state: ViewModelState<Profile>,
+) {
+	if (state.isLoaded()) {
+		UserAvatar(state.data().photo)
```

Comment:
Would be better to call `data()` one time and store it in some variable, or pass it as a parameter to this composable.

Reply:
Done 😄